pkg/scalers/mssql_scaler.go

```diff
@@ -166,7 +168,14 @@ func parseMSSQLMetadata(config *ScalerConfig) (*mssqlMetadata, error) {
 			meta.password = config.ResolvedEnv[config.TriggerMetadata["passwordFromEnv"]]
 		}
 	}
-	meta.metricName = "mssql"
+	switch {
+	case meta.database != "":
+		meta.metricName = kedautil.NormalizeString(fmt.Sprintf("mssql-%s", meta.database))
+	case meta.host != "":
+		meta.metricName = kedautil.NormalizeString(fmt.Sprintf("mssql-%s", meta.host))
+	default:
+		meta.metricName = "mssql"
+	}
```

Comment:
Personally, I liked the option of just having `meta.metricName = "mssql"`

Reply:
Yes. I'm totally fine with that too. By consolidating to a single scenario `meta.metricName = "mssql"` , will it have any foreseeable impact on the end user? 🤔

Reply:
it could, but it shouldn't. I mean, we have deprecated the metric name, and we have suggested using trigger name for external monitoring, if there is any user still using metric name instead of trigger name, they could be affected but it's part of the deprecation process.
If you meant because the metric name in the HPA changes, KEDA reconcilles all the HPAs updating the metric names, so it'll blink, but that's all